src/main/java/uk/gov/hmcts/reform/coh/service/utils/ExpiryCalendar.java

```diff
@@ -1,25 +1,33 @@
 package uk.gov.hmcts.reform.coh.service.utils;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
 import java.util.Calendar;
 import java.util.Date;
 import java.util.GregorianCalendar;
 import java.util.TimeZone;
 
+@Component
 public class ExpiryCalendar {
 
-    private ExpiryCalendar() {}
-    /**
-     * Hard-coded to 7 days.
-     * @return
-     */
+    public static int deadlineExtensionDays;
```

Comment:
Better to have this private otherwise another class could change it

Reply:
I'ts not possible to have spring inject on a static variable like you would a local variable. Hence the hacky workaround

Reply:
Okay. Just make it private then

Reply:
I've changed the class to use INSTANCE and this should encapsulate the deadline days variable properly.